kotsadm/pkg/handlers/troubleshoot.go

```diff
@@ -476,6 +435,7 @@ func GetDefaultTroubleshoot(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte(fullTroubleshoot))
 }
 
+// NOTE: there is no auth on this route
```

Comment:
here

Reply:
i assume this request comes from the `kubectl support-bundle` command. @laverya do you know the answer?

Reply:
yes, this is to be used by `kubectl support-bundle`